Makefile

```diff
@@ -6,7 +6,7 @@ GIR_FILES = gir-files/Gtk-3.0.gir
 gir : src/auto/mod.rs
 
 src/auto/mod.rs : Gir.toml $(GIR) $(GIR_FILES)
-	$(GIR) -c Gir.toml
+	$(GIR) -c Gir.toml -s
```

Comment:
Oh, you meant adding to the makefile. Well, it doesn't seem like something useful in a regular workflow. Isn't the run time in single digits for you now?